shuttle/templates/loaded_views/user_modal.html

```diff
@@ -42,12 +72,31 @@ <h5 class="modal-title">Change role of {{ username }}</h5>
     </div>
 
     <script>
-        $('#edit-modal').on('hidden.bs.modal', function() {
+        $('#delete-modal').on('hidden.bs.modal', function() {
             $('#results').html('');
         });
-        $('#delete-modal').on('hidden.bs.modal', function() {
+        $('#edit-modal').on('hidden.bs.modal', function() {
             $('#results').html('');
         });
+
+        let role = "Driver"
+
+        function addUser(username){
+            $.ajax({
+                type:"POST",
+                url: "{{ url_for('SchedulesView:add_user') }}",
+                data:JSON.stringify({
+                    'username' : username,
+                    'role': role
+                }),
+                contentType:'application/json;charset=UTF-8',
+                success: function(){
+                },
+                error: function(error){
+                }
+            })
+        }
```

Comment:
Either make it go back to the users page or refresh on success since right now nothing happens and it makes me question on whether it succeeded or not.